### ISSUE TITLE
Learning the loss weights as they were parameters

### ISSUE THREAD

**Issue by ogreyesp** [keras-team/keras#12817]
Hello.

As far as I can understand, loss_weights parameter on compile function is used to define how much each of the model output loss contributes to the final loss value.

My question is: Is there a way to learn these loss_weights at the same time with the other parameters of the model?. For example, let say, I have a model with 4 losses, and weights equal to 0.25 initially assigned for each one of the losses, but these weights are automatically being updated (via backpropagation) as if they were parameters of the model.

Is there a possible solution to do this in Keras?

Thanks in advance
